test/wasm/rot.js

```diff
@@ -0,0 +1,14 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft Corporation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+const blob = WScript.LoadBinaryFile('rot.wasm');
+const moduleBytesView = new Uint8Array(blob);
+var a = Wasm.instantiateModule(moduleBytesView, {}).exports;
+print(a.rotl(11,2)); // == 44
```

Comment:
Since you don't seem to use a baseline and you know the result before hand.
Could you make a loop that checks the results and print `SUCCESS` or `FAILED`.
like
```js
const tests = [{op: 'rotl', val: 11, opnd: 2, exp: 44}, ...othertests];
let failed = false;
for(const test of tests) {
  const res = a[test.op](test.val, test.opnd);
  if (res !== test.exp) {
    print(`Error: ${test.op}(${test.val}, ${test.opnd}) returns ${res}, expected ${test.exp}`);
    failed = true;
  }
}
if (!failed) {
  print("SUCCESS");
} else {
  print("FAILED");
}
```